library/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/instrumentation/UserActionTrackingStrategyLegacy.kt

```diff
@@ -29,7 +29,7 @@ internal class UserActionTrackingStrategyLegacy(
 
     override fun onActivityResumed(activity: Activity) {
         super.onActivityResumed(activity)
-        gesturesTracker.startTracking(activity.window, activity)
+        sdkCore?.let { gesturesTracker.startTracking(activity.window, activity, it) }
```

Comment:
how can we protect us from the situation when it won't be executed and we don't know about it?